src/components/MDXRenderer/ExampleBlock/ExampleBlock.scss

```diff
@@ -15,43 +15,16 @@ $block: '.#{variables.$ns}example-block';
         margin-bottom: 0;
     }
 
-    &__theme-root.yc-root {
-        background: var(--g-color-base-background);
-        color: var(--g-color-text-primary);
-
-        @include themes.g-theme-common;
-
-        &_theme_light {
```

Comment:
light theme doesn't work
<img width="851" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/ca05c4d1-2d65-4875-a497-c315865b88f4">

Reply:
How to reproduce? In dev mode it works

Reply:
Hmm, I can reproduce in dev mode

https://github.com/gravity-ui/landing/assets/29424746/0b593646-3c28-4810-8879-1933cb32e567